modules/experimental-layers/src/tile-layer/tile-layer.js

```diff
@@ -0,0 +1,89 @@
+import {GeoJsonLayer, CompositeLayer} from 'deck.gl';
+import TileCache from './utils/tile-cache';
+
+const defaultLineColor = [0, 0, 0, 255];
+const defaultFillColor = [0, 0, 0, 255];
+
+const defaultProps = {
+  stroked: true,
```

Comment:
Since the sub layers are not necessarily GeoJsonLayers, we should not presume the props here. The only two default props needed are `getTileData` and `renderSubLayers`